load.go

```diff
@@ -69,14 +69,20 @@ func (om *OpenMock) populateBehaviors(mocks []*Mock) {
 	for i := range mocks {
 		m := mocks[i]
 		m.loadFile(om.TemplatesDir)
-		if m.Include != "" {
-			m = r.Behaviors[m.Include].patchedWith(*m)
-		}
 		if r.Behaviors[m.Key] != nil {
-			logrus.WithField("key", m.Key).Warn("Multiple Behaviors with same key found. Results can be unexpected.")
+			logrus.WithField("key", m.Key).Info("Overriding existing behavior")
```

Comment:
should this still be `Warn`?

Reply:
My thinking in dropping it back down to an `Info`, is that this is unsurprising behavior compared to previously, attempting to override a named Behavior would actually result in both versions' Actions being performed.